### PR TITLE
Add integration test for the query macro

### PR DESCRIPTION
**Pull request by mamcx**
# Description of Changes

The integration test for `rust-wasm-test` contains the use of the `query!` macro but was not called anywhere.

This later will be used to verify the use of `multi-column` indexes on this macro when https://github.com/clockworklabs/SpacetimeDB/pull/267 lands.

# API and ABI

 - [ ] This is a breaking change to the module ABI
 - [ ] This is a breaking change to the module API
 - [ ] This is a breaking change to the ClientAPI
 - [ ] This is a breaking change to the SDK API

*If the API is breaking, please state below what will break*

# Expected complexity level and risk

1

### CODE REVIEW

File: modules/rust-wasm-test/src/lib.rs [hunk 1]
```diff
@@ -117,10 +118,37 @@ pub fn test(ctx: ReducerContext, arg: TestAlias, arg2: TestB, arg3: TestC) -> an
...
+
+    log::info!("Row count filtered by multi-column condition: {:?}", multi_row_count);
+
     log::info!("END");
     Ok(())
 }
 
+#[spacetimedb(reducer)]
+pub fn test_query(ctx: ReducerContext) -> anyhow::Result<()> {
+    test(
```
Comment (RReverser) [inline modules/rust-wasm-test/src/lib.rs:140]: Do you need both to be reducers? If you want to just invoke this one, then maybe `test` itself shouldn't have `spacetimedb(reducer)`. Alternatively, just invoke `test` itself from standalone_integration_test.

Comment (mamcx) [reply modules/rust-wasm-test/src/lib.rs:140]: I don't understand the purpose of having `test` and not using it elsewhere, I think we speculate it just was to confirm it compiles. If that is the case having both makes some sense.

File: modules/rust-wasm-test/src/lib.rs [hunk 2]
```diff
@@ -117,10 +118,37 @@ pub fn test(ctx: ReducerContext, arg: TestAlias, arg2: TestB, arg3: TestC) -> an
...
+    log::info!("MultiColumn");
+
+    for i in 0i64..1000 {
+        Point::insert(Point {
+            x: i + arg.x as i64,
+            y: i + arg.y as i64,
+        });
+    }
+
+    let multi_row_count = query!(|row: Point| row.x >= 0 && row.x <= 200).count();
```
[the remark this answers is not in the document]
Comment (mamcx) [reply modules/rust-wasm-test/src/lib.rs]: Oh yeah, good catch

File: modules/rust-wasm-test/src/lib.rs (hunk 1 above)
Comment (RReverser) [reply modules/rust-wasm-test/src/lib.rs:140]: Yeah but then you can probably just call `test` directly from your test runner, just pass non-empty `args` maybe?

Comment (mamcx) [reply modules/rust-wasm-test/src/lib.rs]: Done

Comment (kulakowski) [reply modules/rust-wasm-test/src/lib.rs]: You happy with this @RReverser ?

Comment (RReverser) [reply modules/rust-wasm-test/src/lib.rs]: Yup, thanks.

**Review by kulakowski — APPROVED** [review]

**Pull request merged**